SPECS/containerized-data-importer/containerized-data-importer.spec

```
@@ -18,7 +18,7 @@
 Summary:        Container native virtualization
 Name:           containerized-data-importer
 Version:        1.51.0
-Release:        3%{?dist}
+Release:        4%{?dist}
 License:        ASL 2.0
```

Small nit- recently, we decided to start using SPDX short license identifiers rather than Fedora's short names. Can we update this to be [`Apache-2.0`](https://spdx.org/licenses/Apache-2.0.html), instead?

```suggestion
License:        Apache-2.0
```

---

Ok, will be done!